Samples/CustomTensorization/CustomTensorization/CustomTensorization.vcxproj

```diff
@@ -8,7 +8,7 @@
     <ProjectGuid>{4420ecd4-ad04-4183-9879-7d52039e5422}</ProjectGuid>
     <Keyword>Win32Proj</Keyword>
     <RootNamespace>CustomTensorization</RootNamespace>
-    <WindowsTargetPlatformVersion Condition=" '$(WindowsTargetPlatformVersion)' == '' ">10.0.17763.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion Condition=" '$(WindowsTargetPlatformVersion)' == '' ">10.0.18362.0</WindowsTargetPlatformVersion>
     <WindowsTargetPlatformMinVersion>10.0.17134.0</WindowsTargetPlatformMinVersion>

```

Comment:
I wonder why some of the samples set min version to be 17134 instead of 17763?

Reply:
This Sample works for OS low to 17134, the mininum version is the minumum requirement, I think it should be okay.

---
In reply to: [409808882](https://github.com/microsoft/Windows-Machine-Learning/pull/310#discussion_r409808882) [](ancestors = 409808882)

Reply:
17134 if RS4, and wasn't winml release under a different preview namespace at that time? If so that would break building these samples.

Reply:
Hmm, I believe when these sample were created, they should be tested to build against RS4, I don't have 17134 installed. But I think it's also okay to change it to 17763. I'll do that.

---
In reply to: [409814617](https://github.com/microsoft/Windows-Machine-Learning/pull/310#discussion_r409814617) [](ancestors = 409814617)

Reply:
Okay sounds good. Someone may have updated the namespace but forgot to update the min version since they had a more recent SDK installed.

Reply:
Yeah, for example back in RS4 it was a LearningModelPreview instead of LearningModel